workflow/workflow_commands.go

```diff
@@ -1577,7 +1577,7 @@ func findWorkflowStatusValue(name string) (enumspb.WorkflowExecutionStatus, bool
 // historyEventToString convert HistoryEvent to string
 func historyEventToString(e *historypb.HistoryEvent, printFully bool, maxFieldLength int) string {
 	data := getEventAttributes(e)
-	return stringify.AnyToString(data, printFully, maxFieldLength, dataconverter.CustomDataConverter())
+	return stringify.AnyToString(data, printFully, maxFieldLength, converter.GetDefaultDataConverter())
```

Comment:
Correctness: the event being formatted was obtained via the history table iterator https://github.com/temporalio/cli/blob/80a17b4de02405e0edc646754b7d48094bf51095/workflow/workflow_commands.go#L273 

The only place the history table iterator is constructed is in `printWorkflowProgress`, which we have [already established](https://github.com/temporalio/cli/pull/384/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#r1406087982) uses the gRPC interceptor.